lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2547,6 +2548,26 @@ def _add_image_features(self, node_space: schema.NodeSpace) -> None:
         else:
             ...
 
+    def _check_image_capability(self, node_space: schema.NodeSpace) -> None:
+        is_an_supported: Optional[bool] = None
+        azure_runbook = node_space.get_extended_runbook(AzureNodeSchema, AZURE)
+        if azure_runbook.vhd:
+            is_an_supported = azure_runbook.vhd.is_accelerated_network_supported
+
+        # Network interface feature is handled by node_space directly.
+        # Don't need to add in featuers
+        if isinstance(is_an_supported, bool) and not is_an_supported:
```

Comment:
It doesn't need to check twice. Just check like, `if azure_runbook.vhd.is_accelerated_network_supported is not None`

Reply:
Done